README.md

```diff
@@ -478,6 +479,57 @@ final class ExampleTest extends TestCase
 }
 ```
 
+## Assert object
+
+Assert object pattern helps write more readable assert sections. Instead of using a few asserts, we can just prepare an abstraction,
+and use natural language to describe what result is expected.
+
+```php
+final class ExampleTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function example_test_with_asserter(): void
+    {
+        $currentTime = new DateTimeImmutable('2022-11-10 20:00:00');
+        $sut = new OrderService();
+
+        $order = $sut->create($currentTime);
+
+        OrderAsserter::assertThat($order)
+            ->wasCreatedAt($currentTime)
+            ->hasTotal(6000);
+    }
+}
+```
+
+```php
+use PHPUnit\Framework\Assert;
+
+final class OrderAsserter
+{
+    public function __construct(private readonly Order $order) {}
+
+    public static function assertThat(Order $order): self
+    {
+        return new OrderAsserter($order);
+    }
+
+    public function wasCreatedAt(DateTimeImmutable $createdAt): self
```

Comment:
Shouldn't be static as well? 🤔

Reply:
No, we're operating on the state of this object:
```php 
public function wasCreatedAt(DateTimeImmutable $createdAt): self
    {
        Assert::assertEquals($createdAt, $this->order->createdAt);
        return $this;
    }
```

In this case, only assertThat is static, because it's just a named constructor.

Reply:
Ah, right thx 👍🏻